code/controllers/Processes/night_lighting.dm

```diff
@@ -28,10 +28,11 @@
 
 /datum/controller/process/night_lighting/setup()
 	name = "night lighting controller"
-	schedule_interval = 100 // every 5 seconds
```

Comment:
This is every 5 minutes and 50 seconds

Also the previous interval was every 10 seconds
you guys cannot into math